src/components/redux/categories/categories.js

```diff
@@ -1,16 +1,12 @@
-const CHECK_STATUS = 'books/categories/CHECK_STATUS';
+const STATUS_CHEKCKED = 'STATUS_CHEKCKED';
 
-const initialState = [];
-
-export const categoryReducer = (state = initialState, { type }) => {
-  switch (type) {
-    case CHECK_STATUS:
-      return 'Under the construction';
+const categoriesReducer = (state = '', action) => {
+  switch (action.type) {
+    case STATUS_CHEKCKED:
+      return 'Under construction';
     default:
       return state;
   }
 };
-
-export const addStatus = () => ({
-  type: CHECK_STATUS,
-});
+export const checkStatus = () => ({ type: STATUS_CHEKCKED });
+export default categoriesReducer;
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻  however you should create A button for checking the status, this button would use a function `checkStatus`  made in the component (the name may be different)
  * `checkCategoriesStatus` dispatches the `underConstruction` action creator (name of the function can be different)